db/schema.rb

```diff
@@ -310,6 +310,7 @@
     t.string "webauthn_id"
     t.string "full_name"
     t.string "totp_seed"
+    t.string "hashed_mfa_recovery_codes", default: [], array: true
```

Comment:
Do you think we should have the `mfa` prefix first? (ie. `mfa_hashed_recovery_codes` ). That way the user fields are associated to mfa (eg. `mfa_level`) are grouped together.